### PR TITLE
GPC-191: Add grafana

### PR DESCRIPTION
**Pull request by oskarwilliams378**

### CODE REVIEW

File: terraform/shared/modules/grafana/cloudfront-logs.tf [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# This is the logging bucket, it doesn't need logs or versioning
+#tfsec:ignore:aws-s3-enable-bucket-logging
+#tfsec:ignore:aws-s3-enable-versioning
+resource "aws_s3_bucket" "cloudfront_logs_bucket" {
+  bucket = "grafana-cloudfront-logs"
```
Comment (ethanmills) [inline terraform/shared/modules/grafana/cloudfront-logs.tf:5]: Are we keeping this?

File: terraform/shared/modules/grafana/ecs-security.tf [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  type              = "egress"
+  protocol          = "tcp"
+  from_port         = 443
+  to_port           = 443
+  cidr_blocks       = ["0.0.0.0/0"]
+  description       = "ECS task egress rule for HTTPS"
+  security_group_id = aws_security_group.ecs_task.id
+}
+
+resource "aws_security_group_rule" "ecs_efs_ingress" {
```
Comment (ethanmills) [inline terraform/shared/modules/grafana/ecs-security.tf:32]: Probably don't need ingress?

**Review by ethanmills — APPROVED** [review]

File: terraform/shared/modules/grafana/cloudfront-logs.tf (hunk 1 above)
Comment (oskarwilliams378) [reply terraform/shared/modules/grafana/cloudfront-logs.tf:5]: Made 7 days

File: terraform/shared/modules/grafana/ecs-security.tf [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  type                     = "ingress"
+  protocol                 = "tcp"
+  from_port                = 3000
+  to_port                  = 3000
+  source_security_group_id = aws_security_group.lb.id
+  description              = "ECS task ingress rule, allow access from LB"
+  security_group_id        = aws_security_group.ecs_task.id
+}
+
+#tfsec:ignore:aws-ec2-no-public-egress-sgr
```
Comment (olevett) [inline terraform/shared/modules/grafana/ecs-security.tf:21]: Why do we need this egress? Is this to get to prometheus?

Comment (oskarwilliams378) [reply terraform/shared/modules/grafana/ecs-security.tf:21]: Yep

**Pull request merged**